### PR TITLE
fix: make decorative dashboard images smaller, add TablePress icon

### PR DESCRIPTION
**Pull request by SteelWagstaff**
This PR provides a fix for https://github.com/pressbooks/pressbooks/issues/3299. Still need to add the actual icon to the heroicons font set as `\f11d` before this can be merged.

### CODE REVIEW

File: assets/src/styles/heroicons.scss [hunk 1]
```diff
@@ -127,3 +127,7 @@ i[class^="pb-heroicons-"]::before, i[class*=" pb-heroicons-"]::before {
 .pb-heroicons-wrench::before {
   content: "\f11c";
 }
+
+.pb-heroicons-tablepress::before {
+  content: "\f11d";
```
Comment (SteelWagstaff) [inline assets/src/styles/heroicons.scss:132]: @arzola we need to add a new icon to the dashicons font with this character reference

Comment (arzola) [reply assets/src/styles/heroicons.scss:132]: I updated the font family and now includes the table cells icon

Comment (SteelWagstaff) [reply assets/src/styles/heroicons.scss:132]: what's the character reference for the new icon?

Comment (SteelWagstaff) [reply assets/src/styles/heroicons.scss:132]: Found it, nevermind

**Pull request merged**